src/studio-header/BrandNav.jsx

```diff
@@ -1,24 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Hyperlink } from '@openedx/paragon';
 
 const BrandNav = ({
   studioBaseUrl,
   logo,
   logoAltText,
-}) => (
-  <a href={studioBaseUrl}>
-    <img
-      src={logo}
-      alt={logoAltText}
-      className="d-block logo"
-    />
-  </a>
-);
+  onNavigate,
+}) => {
+  const handleClick = (e, url) => {
```

Comment:
[question] Can we move it to the utils because this method repeats several times?

Reply:
Refactored, thanks